lib/aiko/led.py

```diff
@@ -158,11 +187,15 @@ def on_led_message(topic, payload_in):
     return True
 
   if payload_in.startswith("(led:dim "):
-    global dim
     tokens = [float(token) for token in payload_in[9:-1].split()]
-    dim = tokens[0]
+    set_dim(tokens[0])
     return True
 
+  # When I send one debug message, this gets printed twice. Why?
+  if payload_in == "(led:debug)":
+    print_dim()
```

Comment:
… but of course, then you'll need to inline it: 
```suggestion
    print("Dim: ", dim)
```

Reply:
if I remove the other one, I'll do this indeed.